# Домашнє завдання № 2

## Частина третя: Аналіз датасет з Kaggle.com

In [2]:
import matplotlib.pyplot as plt
import pandas as pd

#### 3.1. Прочитайте csv файл (використовуйте функцію read_csv)

In [3]:
df = pd.read_csv("bestsellers with categories.csv")

#### 3.2. Виведіть перші п'ять рядків (використовується функція head)

In [4]:
print(df.head())

                                                Name  \
0                      10-Day Green Smoothie Cleanse   
1                                  11/22/63: A Novel   
2            12 Rules for Life: An Antidote to Chaos   
3                             1984 (Signet Classics)   
4  5,000 Awesome Facts (About Everything!) (Natio...   

                     Author  User Rating  Reviews  Price  Year        Genre  
0                  JJ Smith          4.7    17350      8  2016  Non Fiction  
1              Stephen King          4.6     2052     22  2011      Fiction  
2        Jordan B. Peterson          4.7    18979     15  2018  Non Fiction  
3             George Orwell          4.7    21424      6  2017      Fiction  
4  National Geographic Kids          4.8     7665     12  2019  Non Fiction  


#### 3.3. Виведіть розміри датасету (використовуйте атрибут shape)

In [5]:
print(df.shape)

(550, 7)


#### 3.3. Про скільки книг зберігає дані датасет? Відповідь: 550

#### 3.4. Зміна назв колонок

In [6]:
df.columns = ['name', 'author', 'user_rating', 'reviews', 'price', 'year', 'genre']